mslib/msui/qt5/ui_sideview_options.py

```diff
@@ -287,3 +340,48 @@ def retranslateUi(self, SideViewOptionsDialog):
         self.groupBox_4.setTitle(_translate("SideViewOptionsDialog", "Performance"))
         self.cbDrawCeiling.setText(_translate("SideViewOptionsDialog", "draw ceiling altitude"))
         self.btCeilingColour.setText(_translate("SideViewOptionsDialog", "colour"))
+        self.groupBox_5.setTitle(_translate("SideViewOptionsDialog", "Font Sizes"))
```

Comment:
May be name this box title: "Plot Options". Hmm, somehow in topview these are all "Plot Options"

This will be also valid when we add there more features in the future.

Reply:
> May be name this box title: "Plot Options". Hmm, somehow in topview these are all "Plot Options"
> 
> This will be also valid when we add there more features in the future.

Okay. I will change this to "Plot Options."